tests/unit/Manifest/ManifestLoaderTest.php

```
@@ -244,6 +278,62 @@ public function testExternalManifestWithMissingFileThrows(): void
+            ->willReturn('yii2-extensions/bad');
+
+        try {
+            $this->expectException(RuntimeException::class);
+            $this->expectExceptionMessage('must decode to an object');
+
+            (new ManifestLoader(new ManifestSchema()))->load($package, $providerPath);
+        } finally {
+            @unlink("{$providerPath}/scaffold.json");
+            @rmdir($providerPath);
+        }
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 311. (undefined)

(ErrorControlOperator)